README.md

```diff
@@ -1,134 +1,53 @@
-# BlueWallet - A Bitcoin & Lightning Wallet
+# Shroud Wallet
 
-[![GitHub tag](https://img.shields.io/badge/dynamic/json.svg?url=https://raw.githubusercontent.com/BlueWallet/BlueWallet/master/package.json&query=$.version&label=Version)](https://github.com/BlueWallet/BlueWallet)
-[![CircleCI](https://circleci.com/gh/BlueWallet/BlueWallet.svg?style=svg)](https://circleci.com/gh/BlueWallet/BlueWallet)
-[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
-![](https://img.shields.io/github/license/BlueWallet/BlueWallet.svg)
+⚠️ **Work in Progress** - This wallet is currently under active development.
 
-Thin Bitcoin Wallet.
-Built with React Native and Electrum.
+A non-custodial, Bitcoin-only mobile wallet focused on Silent Payments (BIP-352).
 
-[![Appstore](https://bluewallet.io/uploads/app-store-badge-blue.svg)](https://itunes.apple.com/us/app/bluewallet-bitcoin-wallet/id1376878040?l=ru&ls=1&mt=8)
-[![Playstore](https://bluewallet.io/uploads/play-store-badge-blue.svg)](https://play.google.com/store/apps/details?id=io.bluewallet.bluewallet)
+## Related Projects
 
-Website: [bluewallet.io](https://bluewallet.io)
+This wallet is built using the following sister repositories:
 
-Community: [telegram group](https://t.me/bluewallet)
+- **[silent-pay](https://github.com/Bitshala-Incubator/silent-pay)** - Core Silent Payments library implementing BIP-352 protocol
+- **[silent-pay-indexer](https://github.com/Bitshala-Incubator/silent-pay-indexer)** - Indexer service for efficient Silent Payment transaction scanning
 
-* Private keys never leave your device
-* Lightning Network supported
-* SegWit-first. Replace-By-Fee support
-* Encryption. Plausible deniability
-* And many more [features...](https://bluewallet.io/features)
+## Getting Started
 
+### Prerequisites
+- Node.js >= 20
+- npm
+- Android Studio (for Android development)
+- Xcode (for iOS development)
 
-<img src="https://i.imgur.com/hHYJnMj.png" width="100%">
+### Installation
```

Comment:
lets keep the original build instructions. They are correct right?
Atleast iOS instructions are correct.